src/commands/rdp.ts

```diff
@@ -0,0 +1,79 @@
+/** Copyright © 2024-present P0 Security
+
+This file is part of @p0security/cli
+
+@p0security/cli is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 3 of the License.
+
+@p0security/cli is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with @p0security/cli. If not, see <https://www.gnu.org/licenses/>.
+**/
+import { authenticate } from "../drivers/auth";
+import { print2 } from "../drivers/stdio";
+import { rdp } from "../plugins/rdp";
+import { RdpCommandArgs } from "../types/rdp";
+import { getAppName, getOperatingSystem } from "../util";
+import yargs from "yargs";
+
+export const rdpCommand = (yargs: yargs.Argv) =>
+  yargs.command<RdpCommandArgs>(
+    "rdp session <destination>",
+    "Connect to a Windows virtual machine via RDP",
+    (yargs) =>
+      yargs
+        .positional("destination", {
+          type: "string",
+          demandOption: true,
+        })
+        .option("reason", {
+          describe: "Reason access is needed",
+          type: "string",
+        })
+        .option("debug", {
+          type: "boolean",
+          describe: "Print debug information.",
+          default: false,
+        })
+        .option("configure", {
```

Comment:
`--configure` is a Microsoft provided flag that opens the settings page of the RDP client rather than immediately creating a session. This allows the user to enter settings they may prefer before starting the session.

https://learn.microsoft.com/en-us/cli/azure/network/bastion?view=azure-cli-latest#az-network-bastion-rdp